src/mgmt/srv_target.c

```diff
@@ -504,6 +504,8 @@ tgt_vos_preallocate(uuid_t uuid, daos_size_t scm_size, int tgt_nr)
 			break;
 		}
 
+		/** Align to 4K or locking the region based on the size will fail */
+		scm_size = D_ALIGNUP(scm_size, 1ULL << 12);
```

Comment:
minor, maybe use 1ULL << PAGE_SHIFT instead?

Reply:
Probably the right thing to do is to call sysconf(_SC_PAGE_SIZE) to get the size